genienlp/tasks/dialogue_task.py

```diff
@@ -131,3 +131,53 @@ def get_splits(self, root, **kwargs):
         kwargs['train_target'] = 'dst'
         kwargs['e2e_evaluation'] = self.args.e2e_dialogue_evaluation
         return E2EDialogueDataset.return_splits(path=root, make_example=self._make_example, **kwargs)
+
+
+@register_task('error_cls')
+class ErrorClassificationTask(BiTOD):
```

Comment:
A similar problem here. If this is BiTOD-specific, it should have BiTOD in the name, if not, make sure the code is generic enough that works with all dialogue datasets.